src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -86,11 +112,133 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
     expect((await screen.findAllByRole('radio')).length).to.equal(2);
     expect(screen.getByLabelText('Bozeman, MT')).to.exist;
     expect(screen.getByLabelText('Belgrade, MT')).to.exist;
+    expect(screen.baseElement).to.contain.text(
+      'Request a Primary care provider. (Optional)',
+    );
+    expect(screen.baseElement).to.contain.text(
+      'You can request a provider you’d prefer for this appointment. If they aren’t available, we’ll schedule your appointment with a provider close to your home.',
+    );
+    expect(screen.baseElement).to.contain.text('Choose a provider');
 
     // Continue without filling in required fields
     userEvent.click(screen.getByText(/Continue/i));
 
     expect((await screen.findAllByRole('alert')).length).to.equal(2);
     expect(screen.history.push.called).to.be.false;
   });
+
+  it('should display list of providers when choose a provider clicked', async () => {
+    const store = createTestStore(initialState);
+    await setTypeOfCare(store, /primary care/i);
+    await setTypeOfFacility(store, /Community Care/i);
+    const screen = renderWithStoreAndRouter(
+      <CommunityCareProviderSelectionPage />,
+      {
+        store,
+      },
+    );
+
+    // Trigger provider list loading
+    userEvent.click(await screen.findByText(/Choose a provider/i));
+    await waitFor(() => screen.getByText(/your address on file:/i));
```

Comment:
We typically write these like `expect(await screen.findByText(/your address on file:/i)).to.exist;`

Reply:
Fixed